tasks/targets.py

```diff
@@ -219,10 +219,10 @@ def __init__(self, schema, name, obs_table, columns, task):
         obs_table.tablename = '{prefix}{name}'.format(prefix=OBSERVATORY_PREFIX, name=sha1(
             underscore_slugify(self._id).encode('utf-8')).hexdigest())
         self.table = '{schema}.{table}'.format(schema=OBSERVATORY_SCHEMA, table=obs_table.tablename)
+        self.obs_table = obs_table
```

Comment:
Any special reason to change the name?

Reply:
As we are going to expose it (see [here](https://github.com/CartoDB/bigmetadata/pull/386)) I didn't want to follow the private variables naming convention